README.md

```diff
@@ -1,2 +1,397 @@
+
+  ```sh
+  docker-compose up -d
+  ```
+
+    - To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn test -P docker -Dtest-name="test name"
+  ```
+
+    - After tests are complete stop the Selenium grid by running the following command:
+      
+  ```sh
+  docker-compose down
+  ```
+
+- Output
+
+  It will not open a browser on your machine but run the test.
+
+    - Console output
+
+  ```sh
+  ~paste console output for single test!~
+  ```
+
+    - Browser will not be launched here as we are running inside Docker environment.
+
+### Run Parallel Test on Docker
+
+In this section we will run some tests on a browser on a selenium grid hosted in docker in parallel across multiple browsers
+
+- How to run the test?
+
+    - Start the Selenium Grid first by running the following command:
+
+  ```sh
+  docker-compose up -d
+  docker-compose scale chromenode=3
+  ```
+
+    - To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn test -P docker-parallel
+  ```
+
+    - After the tests are complete stop the Selenium grid by running the following command:
+
+  ```sh
+  docker-copmpose down
+  ```
+
+- Output
+
+  It will not open a browser on your machine but run the test.
+
+    - Console output
+
+  ```sh
+  ~paste console output for parallel test!~
+  ```
+
+    - Browser will not be launched here as we are running inside Docker environment.
+
+---
+
+# BrowserStack
+
+App & Browser Testing Made Easy!! Give your users a seamless experience by testing on 2000+ real devices and browsers. Don't compromise with emulators and simulators.
+
+## Prerequisites
+
+- Create an account on BrowserStack if not already
+- Export the environment variables for the Username and Access Key of your BrowserStack account.
+
+    - For \*nix based and Mac machines:
+
+  ```sh
+  export BROWSERSTACK_USERNAME=<browserstack-username> &&
+  export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+    - For Windows:
+
+  ```shell
+  set BROWSERSTACK_USERNAME=<browserstack-username>
+  set BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+- Clone the browserstack-canonical-app repository using `git clone https://github.com/browserstack/browserstack-demo-app`
+- Update the [caps.json](resources/conf/caps/caps.json) file according to your device/browser needs. You can find the capabilities needed from here: [browserstack-capability-generator](https://browserstack.com/automate/capabilities)
+
+## Run Your Tests
+
+In this section we will run tests on BrowserStack infrastructure
+
+### Run Single Test on Browserstack
+
+In this section we will run a single the test on chrome on Browserstack. To change Capabilities for this configuration refer to `single` object in `caps.json` file.
+
+- How to run the test?
+
+  To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn install -P bstack-single -Dtest-name="test name"
```

Comment:
https://automate.browserstack.com/dashboard/v2/builds/605f0a2e0c8ab3711d39d69c6af945323e42eb94/sessions/5409d5495ef91fb38e02955f08a587bdf38fe7c8

Looking for "Orders" link text leads to the test failing

Reply:
fixed